### PR TITLE
[PAF-6]Khoa/paf 6

### PR DESCRIPTION
**Pull request by NguyennKhoaa**

### CODE REVIEW

File: src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx [hunk 1]
```diff
@@ -229,50 +213,20 @@ const TableUser = (): JSX.Element => {
...
-                </ButtonBase>
-              ) : (
-                <ButtonBase
-                  sx={{
-                    fontSize: '16px',
-                    width: '100px',
-                    height: '50px',
-                  }}
-                  onClick={(): void => {
-                    setSort('ASC');
```
Comment (baosphoa0403) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx]: revert

File: src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx [hunk 2]
```diff
@@ -287,20 +241,20 @@ const TableUser = (): JSX.Element => {
...
                   <TableCell align="left">
                     <Avatar
                       {...stringAvatar(item.fullName)}
                       src={item.avatar}
                     />
                   </TableCell>
+                  <TableCell align="left">{item.username}</TableCell>
+                  <TableCell align="left">{item.fullName}</TableCell>
+                  {/* <TableCell align="left">{item.firstName}</TableCell>
+                  <TableCell align="left">{item.lastName}</TableCell> */}
```
Comment (baosphoa0403) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx]: remove

File: src/pages/DashboardAdmin/SlideBar/SlideBar.tsx [hunk 3]
```diff
@@ -71,10 +69,6 @@ const Sidebar = ({
               navigate(PATH_NAME.DashboardAdminPayment);
             }}
           >
-            <ListItemIcon>
-              <AccountBalanceWalletIcon sx={{ fontSize: '32px' }} />
-            </ListItemIcon>
-            <ListItemText>Wallet</ListItemText>
```
Comment (baosphoa0403) [inline src/pages/DashboardAdmin/SlideBar/SlideBar.tsx]: revert here

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 4]
```diff
@@ -158,27 +130,6 @@ export default function FormAddUser({
...
-                      'DOB',
-                      new Date(newValue as string).toLocaleDateString()
-                    );
-                  }}
-                  renderInput={(params): JSX.Element => (
-                    <TextField className={classes.root} {...params} />
-                  )}
-                />
-              </LocalizationProvider>
-            </Box> */}
```
Comment (baosphoa0403) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx:181]: Update DOB

**Review by baosphoa0403 — CHANGES_REQUESTED** [review]

**Review by baosphoa0403 — APPROVED** [review]

**Review by lpduy2k — APPROVED** [review]

**Pull request merged**